build_rust/README.md

```diff
@@ -0,0 +1,31 @@
+The Rust Build System
+=====================
+
+The Rust Build System is written in Rust and is managed as a Cargo package.
+Building the latest Rust compiler is as simple as running
```

Comment:
there should be a space after this paragraph and before the triple backticks, same everywhere else

also, a `:` at the end of this line would be nice, keep it consistent with the others